core/src/main/java/com/scalar/db/transaction/consensuscommit/CoordinatorException.java

```diff
@@ -1,6 +1,6 @@
-package com.scalar.db.exception.transaction;
+package com.scalar.db.transaction.consensuscommit;
 
-public class CoordinatorException extends TransactionException {
+public class CoordinatorException extends Exception {
```

Comment:
I also changed `CoordinatorException` to extend from `TransactionException` to `Exception`.